cocos/3d/reflection-probe/reflection-probe-component.ts

```diff
@@ -105,10 +105,6 @@ export class ReflectionProbe extends Component {
 
     private _position = new Vec3(0, 0, 0);
 
-    constructor () {
-        super();
-    }
```

Comment:
Removing the default constructor will make the package size bigger. So don't remove it.